.eslintrc.js

```diff
@@ -167,6 +168,12 @@ module.exports = {
                 react: {
                     version: 'detect',
                 },
+                'import/resolver': {
+                    node: {
+                        extensions: ['.ts', '.tsx'],
```

Comment:
don't we have this for js and jsx files?

Reply:
.js is the default for the resolver as per their document. I guess it is ok to skip that here.